markdown/ch1.markdown

```diff
@@ -305,7 +313,7 @@ y = np.reshape(y, (1, len(y)))
 print(y)
 ```
 
-In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match (or be equal to 1).
+In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match or be equal to 1.
```

Comment:
This is false.

The arrays need not have the same number of dimensions, and either dimension need to be equal to None or 1 or they must match.